test/augmentation/test_container.py

```diff
@@ -308,8 +308,8 @@ def test_forward_and_inverse(self, random_apply, return_transform, device, dtype
             torch.tensor([[[155, 0], [900, 0], [900, 400], [155, 400]]], device=device, dtype=dtype), 1000, 500
         )[:, None].float()
         aug = K.AugmentationSequential(
-            K.ImageSequential(
-                K.ColorJitter(0.1, 0.1, 0.1, 0.1, p=1.0), K.RandomAffine(360, p=1.0, return_transform=True)
+            K.AugmentationSequential(
```

Comment:
I'd keep `ImageSequential` to make sure we don't break anything on that end. Just add your proposal in a extra linte.